cumulusci/cumulusci.yml

```diff
@@ -314,6 +314,10 @@ tasks:
     description: Creates a Github release for a given managed package version number
     class_path: cumulusci.tasks.github.CreateRelease
     group: GitHub
+  gather_release_notes:
+    description: Generates release notes by parsing pull request bodies of merged pull requests between two tags
```

Comment:
This description is the same as `github_release_notes`, what is this task doing differently between the two?